### PR TITLE
Implemented multi framework targeting

### PR DESCRIPTION
**Pull request by sglogovic**
XCellKit now supports multiple framework versions - 4.0 , 4.5 and 4.5.2 

I have tested and it works for both 4.5 and 4.5.2, but I'm not sure how build server will handle this change.

### CODE REVIEW

File: Acceleratio.XCellKit/Acceleratio.XCellKit.csproj [hunk 1]
```diff
@@ -41,43 +41,14 @@
...
-    <Compile Include="SpredsheetHyperlinkManager.cs" />
-    <Compile Include="SpredsheetLocation.cs" />
-    <Compile Include="SpredsheetRow.cs" />
-    <Compile Include="SpredsheetStyle.cs" />
-    <Compile Include="SpredsheetStylesManager.cs" />
-    <Compile Include="SpredsheetTable.cs" />
-    <Compile Include="SpredsheetTableColumn.cs" />
-    <Compile Include="SpredsheetWorkbook.cs" />
-    <Compile Include="SpredsheetWorksheet.cs" />
+    <None Include="packages.config" />
```
Comment (vinkobedek) [inline Acceleratio.XCellKit/Acceleratio.XCellKit.csproj:48]: i see this file included here, but i dont see it in the list of files

Comment (vinkobedek) [reply Acceleratio.XCellKit/Acceleratio.XCellKit.csproj:48]: sorry. my bad, this is just location changing.

File: Acceleratio.XCellKit/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,36 +0,0 @@
...
-//
-//      Major Version
-//      Minor Version 
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers 
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (vinkobedek) [inline Acceleratio.XCellKit/Properties/AssemblyInfo.cs:35]: where does the dll get it's version from now, if this file is deleted?

Comment (sglogovic) [reply Acceleratio.XCellKit/Properties/AssemblyInfo.cs:35]: Acceleratio.XCellKit Project properties now has additional tab called Package which has all this information

image

**Review by vinkobedek — APPROVED** [review]

**Pull request merged**